### PR TITLE
VIH-9510 Witness Interpreter VMR cannot be called into hearing

### PR DESCRIPTION
**Pull request by oliver-scott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9510

### Change description ###
Fixes an issue where if a witness joins a conference, the conference is paused, and then an interpreter is added, the witness + interpreter cannot be called into the conference.

- Upon a participant being added to a room, add their linked participants who have joined the conference but are not already in the room

- Replace hmcts.net emails with email.com in tests (security)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/vh-video-api/values.dev.template.yaml [hunk 1]
```diff
@@ -2,4 +2,7 @@
 java:
   image: '${IMAGE_NAME}'
   ingressHost: ${SERVICE_FQDN}
-  releaseNameOverride: ${RELEASE_NAME}
\ No newline at end of file
+  releaseNameOverride: ${RELEASE_NAME}
```
Comment (oliver-scott) [inline charts/vh-video-api/values.dev.template.yaml:5]: TODO revert these changes before merging

**Review by will-craig — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Review by will-craig — APPROVED** [review]
